.circleci/config.yml

```diff
@@ -15,4 +15,4 @@ jobs:
       - run: go fmt ./...
       - run: go vet ./...
       - run: CGO_ENABLED=0 go test -mod=vendor ./cmd/... -count=1 -p=1
-      - run: CGO_ENABLED=0 go test -mod=vendor ./pkg/... -count=1
+      - run: CGO_ENABLED=0 go test -mod=vendor ./pkg/... -count=1 -p=1
```

Comment:
Are we having concurrency problems in the `pkg` tests as well? :disappointed:

Reply:
yes